app/controllers/api/v1/games_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
         end
 
         game = Game.create(player_1: Player.new(Board.new(4), player_1.apikey), player_2: Player.new(Board.new(4), player_2.apikey))
+        # flash[:welcome] = "Ahoy! You're playing on a 4x4 grid and you have 2 ships of lengths 3 and 2 to place"
```

Comment:
Lets not forget this! I'll add to pivotal to send this as another email. Not a priority, but I think good UX.